capstone-app/src/app/modules/maps/map/map.component.spec.ts

```diff
@@ -54,8 +38,8 @@ describe('MapComponent', () => {
       lat++;
       lng++;
       fixture.componentInstance.markers.push(coordinates);
-      fixture.detectChanges();
     }
+    fixture.detectChanges();
 
     const markerDebug = fixture.debugElement.queryAll(By.css('.marker'));
     const markerComponent = markerDebug.map(marker => marker.componentInstance.getPosition());
```

Comment:
This variable name indicates that it's a single component, instead of multiple coordinates. Rename "markerPositions" or something.